config/crd/bases/fence-agents.medik8s.io_fenceagentsremediations.yaml

```diff
@@ -5,9 +5,9 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.8.0
   creationTimestamp: null
-  name: fenceagentsremediations.fence-agents.medik8s.io
+  name: fenceagentsremediations.fence-agents-remediation.medik8s.io
```

Comment:
I think this file should be deleted, there is a new one

Reply:
Correct :)